server/routerlicious/packages/protocol-base/package.json

```diff
@@ -59,9 +59,9 @@
 		"temp-directory": "nyc/.nyc_output"
 	},
 	"dependencies": {
-		"@fluidframework/common-utils": "^3.1.0",
 		"@fluidframework/gitresources": "workspace:~",
 		"@fluidframework/protocol-definitions": "^3.2.0",
+		"@types/events": "^3.0.0",
```

Comment:
Not sure if this is really necessary, since it's a `@types` package. It's there in common-utils.